src/GUI/RevEng.Shared/Cli/Configuration/FileLayout.cs

```diff
@@ -17,5 +17,9 @@ public class FileLayout
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
         [JsonPropertyName("use-schema-folders-preview")]
         public bool UseSchemaFoldersPreview { get; set; }
+
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        [JsonPropertyName("use-schema-namespaces-preview")]
```

Comment:
Add this option to the json schema file in the samples folder

Reply:
Do you mean the json files in the src\GUI\efreveng60\TestFiles folder? I can't find a "samples" folder. Thanks.

Reply:
Sorry, it is this one https://github.com/ErikEJ/EFCorePowerTools/blob/master/samples/efcpt-schema.json